Automattic/wp-calypso
# Rename All In plan instances to Complete

### Changes proposed in this Pull Request

Rename files and slugs to Jetpack Complete. In this PR, this mostly concerns the product icon.

### Testing instructions

- Visit `/devdocs/design/product-icon`
- Check that you can still see the icon in the "Jetpack Plan Icons v2" section
- Visit `/devdocs/design/jetpack-plan-card`
- Check that you still see the icon and that All In has been replaced by Complete

Note: card colours are fixed in a separate PR

### Screenshots

#### ProductIcon
<img width="194" alt="Screen Shot 2020-08-06 at 2 56 45 PM" src="https://user-images.githubusercontent.com/1620183/89571875-5f5bcd80-d7f6-11ea-822b-961fd3ee6c6c.png">

#### JetpackPlanCard
<img width="618" alt="Screen Shot 2020-08-06 at 2 57 18 PM" src="https://user-images.githubusercontent.com/1620183/89571891-6682db80-d7f6-11ea-841a-08a95b169abc.png">
